src/components/CreateUser.vue

```diff
@@ -0,0 +1,159 @@
+<template>
+  <div>
+    <md-card style="padding : 10px;">
+      <md-card-header>
+        <h1 class="md-title">Create new account</h1>
+      </md-card-header>
+
+      <md-field :class="usernameClass">
+        <md-icon>person</md-icon>
+        <label>Username</label>
+        <md-input v-model="username" type="text" v-on:keyup="fromIncorrecUsernameToCorrect()" v-on:blur="isUsernameValid()" required/>
+        <span class="md-error">A username must contain at least five characters</span>
+      </md-field>
+
+      <md-field :class="emailClass">
+        <md-icon>email</md-icon>
+        <label>Email</label>
+        <md-input v-model="email" type="email" v-on:keyup="fromIncorrecEmailToCorrect()" v-on:blur="isEmailValid()" required/>
+        <span class="md-error">Please provide valid email address</span>
+      </md-field>
+
+      <md-field :class="passwordClass">
+        <md-icon>lock</md-icon>
+        <label>Password</label>
+        <md-input v-model="password" type="password" v-on:keyup="fromIncorrecPasswordToCorrect" v-on:blur="isPasswordValid()" required/>
+        <span class="md-error">A password must contain at least eight characters</span>
+      </md-field>
+
+      <div>
+        <label>Permissions:</label>
+        <div>
+          <md-switch class="md-primary" v-model="isModerator">
+            <label>Moderator</label>
+          </md-switch>
+        </div>
+        <div>
+          <md-switch class="md-primary" v-model="isCandidate">
+            <label>Candidate</label>
+          </md-switch>
+        </div>
+        <div>
+          <md-switch class="md-primary" v-model="isRedactor">
+            <label> Redactor</label>
+          </md-switch>
+        </div>
+      </div>
+      <md-button @click="submit_click" class="md-raised md-primary"> {{addUser}}</md-button>
+    </md-card>
+  </div>
+</template>
+
+<script>
+  import firebase from 'firebase'
+  import {config} from "../App"
+  import {db} from "../App"
+
+  const createAccountFirebase = firebase.initializeApp(config, "create_account_instance").auth();
```

Comment:
Jesteś pewien, że to jest odpowiedni sposób? Nie lepiej np używać funkcji i admin sdk do tworzenia konta?

Reply:
Nie widzę lepszego sposobu, a używanie admin sdk raczej jest błędne gdy mamy już usera z prawami do tworzenia konta.